dockstore-webservice/src/main/resources/migrations.1.10.0.xml

```diff
@@ -77,4 +77,12 @@
     <changeSet author="nolwarre (generated)" id="notificationsCharLimit">
         <modifyDataType columnName="message" newDataType="varchar(1024)" tableName="notification"/>
     </changeSet>
+    <changeSet author="nzhao (generated)" id="workflowForumUrl">
+        <addColumn tableName="service">
+            <column name="forumurl" type="varchar(255 BYTE)"/>
```

Comment:
Any danger of getting close to the limit? I don't even know what a forumurl looks like? If it's remotely possible, suggest increasing to 512.

Reply:
I think it could be any URL from a discussion forum like 
https://www.biostars.org/p/464724/
https://gitter.im/ga4gh/dockstore
etc.

Reply:
I think 256 should be enough? Not sure if super long urls for a discussion forum are common, the examples Denis included are pretty short

Reply:
Should be OK then. But note that there is no PG penalty to making it larger (they don't pre-allocate the space).

Reply:
I thought Denis banned us from using varchar instead of text?

Reply:
I think the ban was for `character(n)` or `char(n)` since there is no benefit vs varchar.
https://www.postgresql.org/docs/9.2/datatype-character.html

Reply:
Hmm...I don't think Liquibase auto-generates with char(n) or character(n) so seems unlikely. Now, do I care enough to dig up an old PR. hmm...

Reply:
https://github.com/dockstore/dockstore/pull/2651#discussion_r309345944

Reply:
To be clear, what I meant, and think/hope is true, was that VARCHAR(1024) does not use more space or have a perf impact compared to VARCHAR(256) if the actual value < 256.

Reply:
I went with varchar because I noticed the giturl column is also varchar, and I went with the logic that since both are urls they would be the same

Reply:
I think the character limit works in this case to avoid someone spamming us with a 10MB forum url (so not `text`). 
No real preference between 256 and 512